MvvmCross.iOS.Support/MvvmCross.iOS.Support.csproj

```diff
@@ -83,6 +83,7 @@
     <Compile Include="SidePanels\MvxPanelPopToRootPresentationHint.cs" />
     <Compile Include="SidePanels\MvxPanelPresentationHint.cs" />
     <Compile Include="SidePanels\IMvxSideMenu.cs" />
+    <Compile Include="Views\MvxUIRefreshControl.cs" />
```

Comment:
I often have this construct in my VM:

```
private MvxCommand _derp;

public ICommand Derp
{
    get { return _derp }
}
```

Ofc where _derp is initialized, but you get the idea.